commands/deploy.go

```diff
@@ -249,7 +249,28 @@ Error: %s`, fprocessErr.Error())
 				function.ReadOnlyRootFilesystem = deployFlags.readOnlyRootFilesystem
 			}
 
-			statusCode := proxy.DeployFunction(function.FProcess, services.Provider.GatewayURL, function.Name, function.Image, function.RegistryAuth, function.Language, deployFlags.replace, allEnvironment, services.Provider.Network, functionConstraints, deployFlags.update, deployFlags.secrets, allLabels, annotations, functionResourceRequest1, function.ReadOnlyRootFilesystem, tlsInsecure)
+			deploySpec := &proxy.DeployFunctionSpec{
```

Comment:
How similar is this to the struct we use in the requests package in the faas project?

Reply:
There are similarities but `CreateFunctionRequest` is missing the following attributes

* `Gateway                 string `
* `Language              string` - interestingly doesn't seem to ever be read
* Replace                 bool
* Update                  bool
* TlsInsecure           bool